intelmq_webinput_csv/bin/backend.py

```diff
@@ -404,6 +404,11 @@ def submit():
     successful_lines = 0
 
     raw_header = []
+
+    # Ensure Harmonization config is only loaded once
+    harmonization_config = dict()
+    harmonization_config['event'] = Event().harmonization_config
```

Comment:
```suggestion
    harmonization = load_configuration(HARMONIZATION_CONF_FILE)
```

And at the top of the file:
```python
from intelmq.lib.utils import RewindableFileHandle, load_configuration # the line exists partially, add load_configuration to it
from intelmq import HARMONIZATION_CONF_FILE
```

Reply:
Done